.gitignore

```diff
@@ -2,6 +2,7 @@ node_modules
 LiveReload.safariextension/*.js
 Chrome/LiveReload/*.js
 Firefox/LiveReload/*.js
+Firefox/LiveReload-Dark/*.js
```

Comment:
We can prob roll back this line but I can do that via commandeer

Reply:
Should I remove this line?

Reply:
Yea, if you have the time. Otherwise, I'll do it before landing. I think we no longer have a `LiveReload-Dark` folder